### PR TITLE
Cool looking Hub

### PR DESCRIPTION
**Pull request by Jeff-Moreau**
Still need to add coding for the level entrances
but mostly done
took me a few hours

### CODE REVIEW

File: Content/HubLandscape_Material.uasset
Comment (Saretzky) [inline Content/HubLandscape_Material.uasset:1]: put this in a relevant folder

**Review by Saretzky — CHANGES_REQUESTED** [review]

File: Content/HubLandscape_Material.uasset
Comment (Jeff-Moreau) [reply Content/HubLandscape_Material.uasset:1]: not there on mine
image

Comment (Jeff-Moreau) [reply Content/HubLandscape_Material.uasset:1]: its here on mine
image

Comment (Saretzky) [reply Content/HubLandscape_Material.uasset:1]: weird, in engine it's there but in github its just in the content folder

**Review by Saretzky — CHANGES_REQUESTED** [review]
move/rotate the player start so the camera isnt squished against the wall
from here:
image

to here for example:
image

**Comment by Jeff-Moreau** [comment]
yup weird, did the player start differently

**Review by Saretzky — APPROVED** [review]
looks good

**Review by Guysus — APPROVED** [review]
look really good

**Pull request merged**